requirements.txt

```diff
@@ -1,12 +1,2 @@
 # the default package dependencies
-
-numpy>=1.17.2
-torch>=1.8.*
-tqdm>=4.57.0
-PyYAML>=5.4
-fsspec[http]>=2021.05.0, !=2021.06.0
-tensorboard>=2.2.0
-torchmetrics>=0.4.1
-pyDeprecate>=0.3.1, <0.4.0
-packaging>=17.0
-typing-extensions>=4.0.0
+-r ./requirements/base.txt
```

Comment:
This contains the pinned dependencies, which is probably undesired for the release

Reply:
The pin shall be removed setup, but rather double check that also this is included

Reply:
How do we double-check it?

Reply:
Oh right, I forgot that setup.py does this. 

> How do we double-check it?
I guess we would reduce one of the pinned version to a lower number, install lightning, then try to upgrade that package to the latest and observe whether pip complains about dependency resolution. Might be too much effort though.

Reply:
Checked and it is there...